test/unit/cinstance_test.rb

```diff
@@ -24,60 +21,60 @@ def teardown
   test 'Cinstance.live_at returns cinstances live at given time' do
     cinstance_one = FactoryBot.create(:cinstance)
 
-    Timecop.travel(1.year.ago)
+    travel_to(1.year.ago)
     cinstance_two = FactoryBot.create(:cinstance)
+    travel_back
```

Comment:
```suggestion
    cinstance_two = travel_to(1.year.ago) { FactoryBot.create(:cinstance) }
```

But you have also two other option:
* define `cinstance_two` outside block so when assigned inside it will be also outside
* you can put the assertions inside the block

Reply:
in general in any usage, it should be impossible to forget the time frozen. for example if `FactoryBot.create(:cinstance)` raises for some reason, then nothing will bring time back to normal and other tests ca suffer.

Reply:
Your suggested change is the best solution